src/ucp/core/ucp_context.c

```diff
@@ -336,6 +336,13 @@ static ucs_config_field_t ucp_config_table[] = {
    "endpoint.",
    ucs_offsetof(ucp_config_t, ctx.proto_indirect_id), UCS_CONFIG_TYPE_ON_OFF_AUTO},
 
+  {"RNDV_PUT_FORCE_FLUSH", "n",
+   "When using rendezvous put protocol, force using a flush operation to ensure\n"
+   "remote data delivery before sending ATP message. If disabled, the protocol\n"
+   "will use fence operation and send ATP immediately on the lane used for put,\n"
+   "without waiting for remote completion.",
```

Comment:
imo, it's better to rephrase and mention that UCP will decide automatically (since flush can be used when not all lanes can send atp)